README.md

```diff
@@ -79,22 +79,25 @@ public void csvLineHasCorrectFormat() throws Exception {
   final String fields = "^[^|]+|[^|]+|[^|]+$";
   new Assertion<>(
     "must match the expected pattern",
-    () -> new FirstOf<>(
+    new FirstOf<>(
       line -> true,
-      new SplitText(
+      new Split(
         new TextOf(new File("report.csv")),
         "\n"
-      )
+      ),
+      () -> new TextOf("")
     ).value(),
     new MatchesRegex(fields)
   ).affirm();
 }
 
 @Test
-public void textIsBlank() {
+public void textIsBlank() throws IOException {
```

Comment:
@fanifieiev  This code won't work: `TextOf.asString` throws an `IOException` which is not being handled in this example. Also, you can remove the `<>` from `Assertion`

Reply:
@paulodamaso Why removing `<>` from `Assertion`? The `Assertion` is a generic type.

Reply:
@fanifieiev Right